runtime/src/test/java/org/corfudb/protocols/wireprotocol/ICorfuPayloadTest.java

```diff
@@ -0,0 +1,53 @@
+package org.corfudb.protocols.wireprotocol;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.corfudb.protocols.logprotocol.CheckpointEntry.CheckpointEntryType;
+import org.corfudb.protocols.wireprotocol.IMetadata.DataRank;
+import org.corfudb.runtime.view.Layout;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+
+public class ICorfuPayloadTest {
```

Comment:
Can you extend AbstractCorfuTest to be uniform with the rest of the framework?

Reply:
I think that's okay since I believe you are moving the test to their specific modules.